netbox/dcim/models/mixins.py

```diff
@@ -1,56 +1,12 @@
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.utils.translation import gettext_lazy as _
-from dcim.choices import *
-from utilities.conversion import to_grams
 
 __all__ = (
     'RenderConfigMixin',
-    'WeightMixin',
 )
 
 
-class WeightMixin(models.Model):
```

Comment:
Is there a reason to move this right now? We shouldn't be touching anything not directly related to the distance fields in this PR, as it's not relevant to the feature request. (We can reorganize the weight resources under a separate PR if needed.)